library/init/meta/mk_has_reflect_instance.lean

```diff
@@ -0,0 +1,73 @@
+/-
+Copyright (c) 2017 Microsoft Corporation. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Sebastian Ullrich
+
+Helper tactic for constructing has_reflect instance.
+-/
+prelude
```

Comment:
Could you please add comments describing what is generated?
That is, the general form of the generated instances.